src/Logger.js

```diff
@@ -60,6 +76,7 @@ class Logger {
 
   succeed() {
     this.loader.succeed();
+    Logger.summary();
```

Comment:
If that's called here, it'll be shown for each element.

`Logger.summary()` should be called inside cli.js in the end.

You can use `yarn babel-node src/cli.js <path_to_more_elements>` or `node_modules/.bin/babel-node src/cli.js <path_to_more_elements>` to test against more elements.